modules/View.js

```diff
@@ -52,10 +52,43 @@
       this.feedbackModel = new Cell();
       this.__childViews = {};
       this.__feedbackEvents = [];
+      this.__applyTimingEventListeners(['pre-dom-event-callback', 'post-dom-event-callback', 'pre-render', 'post-render']);
```

Comment:
if these are only for timing events, maybe we could simplify it and say that you pass in 'dom-event-callback' and 'render' and we do the prefixing?